src/components/Marketplace/MarketplaceInput.js

```diff
@@ -0,0 +1,241 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import { Container } from '@material-ui/core';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import MarketplaceInputField from './MarketplaceInputField';
+
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base('appBsavky1VJ0HK23');
+const today = new Date().toDateString();
+export default function MarketplaceInput() {
+  const initialState = {
+    crop: '',
+    description: '',
+    'unit type': '',
+    'units per pallet': 0,
+    'lbs per unit': 0.0,
+    'standard price per unit': 0.0,
+    'expiration date': '',
+    'first available date': '',
+    'date entered': today,
+    'available until': '',
+    'growing season': '',
+    'units available': 0,
+    'units pending': 0,
+    'units sold': 0,
+    distressed: false,
+  };
+  const [marketplaceRecord, setMarketplaceRecord] = useState(initialState);
+  function resetState() {
+    setMarketplaceRecord({ ...initialState });
+  }
+
+  function createRecord(e) {
```

Comment:
Right now, this function resets the state when you press submit, but in our front-end it doesn't clear all the inputs to be their default values. Do you think you could explore trying to reset the form to it's initial state front end wise? Don't spend more than 30 minutes on this, and if you cannot figure it out, just write a TODO comment in this function like

/* TODO: reset the form on the front end so that it reflects its initial state when an individual presses submit */